types/src/model/blockchain.rs

```diff
@@ -488,3 +498,72 @@ pub struct GetTxOutSetInfo {
 /// Models the result of JSON-RPC method `verifytxoutproof`.
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
 pub struct VerifyTxOutProof(pub Vec<Txid>);
+
+/// Models the result of the JSON-RPC method `getdescriptoractivity`.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct GetDescriptorActivity {
+    /// A list of activity events related to the descriptors.
+    pub activity: Vec<ActivityEntry>,
+}
+
+/// Models a script pubkey with strongly typed fields.
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct ScriptPubkey {
```

Comment:
I don't think we want this. We have a bunch of useage of this type already. Are you able to do the same in this PR. I.e., convert the data out of this to the correct rust-bitcoin types.

Reply:
I didn't code a solution but I'm reasonable confident that this type should not exist. See what was done in #174.